aws/cloud-formation/codeArtifact/README.md

```diff
@@ -0,0 +1,100 @@
+# CodeArtifact Setup Guide
+
+## Introduction
+
+The Viite project uses AWS CodeArtifact to store and manage our npm and Maven packages. 
+
+**Note for Local Development:**
+While CodeArtifact is used for our CI/CD pipeline, its use is optional for local development. You can choose to develop using public package repositories if you prefer. 
+
+This guide explains how to set up CodeArtifact for both npm (frontend) and SBT/Maven (backend) dependencies, should you choose to use it for your local development.
+
+## Prerequisites
+
+- AWS CLI installed and configured
+- Node.js and npm installed (for frontend)
+- SBT installed (for backend)
+
+**Note**: The AWS CodeArtifact AuthToken is valid for 12 hours. You will need to redo next steps every time you choose to use CodeArtifact for packages.
```

Comment:
This comment is good!
But now it gets no  attention when just skimming through.
It is under prerequisites, but is distinct from those.

Hmm, coupling the login, and setup paragraphs under e.g. 
"## Taking CodeArtifact locally in use"
and changing their header style to ###
And then make the Note the first thing under "## Taking CodeArtifact locally in use"?